README.md

```diff
@@ -65,10 +65,10 @@ usually take place the last Wednesday of the month at 9am Pacific / 4pm UTC.
 
 We're a friendly group, so please feel free to join us!
 
-## Code of conduct
+## Code of Conduct
 
 This project is for everyone. We ask that our users and contributors take a few
-minutes to review our [code of conduct][coc].
+minutes to review our [Code of Conduct][CoC].
```

Comment:
I'm willing to drop any family of changes.

Do note that formally you follow https://github.com/cncf/foundation/blob/master/code-of-conduct.md which uses the case I'm suggesting.